### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jparsana**
CoC for OpenCenus as agreed to in previous Community Meeting

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+census-developers@googlegroups.com
```
Comment (SergeyKanzhelev) [inline code-of-conduct.md]: what does this top line with an e-mail represent?

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# OpenCensus Code of Conduct 
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of OpenCensus pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation. People violating this code of conduct may be banned from the community and all its events.
+
+## Our Standards and Values
+
+- Be honest:
```
Comment (SergeyKanzhelev) [inline code-of-conduct.md]: why ends with semicolon?

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Be friendly and patient: Remember you might not be communicating in someone else's primary spoken or programming language, and others may not have your level of understanding.
+- Be welcoming: Our communities welcome and support people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+- Be respectful: We are a world-wide community of professionals, and we conduct ourselves professionally. Disagreement is no excuse for poor behavior and poor manners. Disrespectful and unacceptable behavior includes, but is not limited to:
+	- Violent threats or language.
```
Comment (SergeyKanzhelev) [inline code-of-conduct.md]: use either spaces or tabs. Please do not combine

File: code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at *census-developers@googlegroups.com* . All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project’s leadership.
+
+## Attribution
+
+*This code of conduct is based the [Contributor Covenant version 1.4](http://contributor-covenant.org/version/1/4/). The ‘Our Standards’ section is based off of the Microsoft Inc’s [Code of Conduct*](https://opensource.microsoft.com/codeofconduct/)
```
Comment (SergeyKanzhelev) [inline code-of-conduct.md]: I'm not sure we need `*` (italic). If it needed - please finish italic after last url

File: code-of-conduct.md (hunk 1 above)
Comment (jparsana) [reply code-of-conduct.md]: I think it accidentally got in there. Will be removing it.

File: code-of-conduct.md (hunk 2 above)
Comment (jparsana) [reply code-of-conduct.md]: Typo.. its been removed till we have a section on honesty.

File: code-of-conduct.md (hunk 3 above)
Comment (jparsana) [reply code-of-conduct.md]: corrected.

**Comment by jparsana** [comment]
@SergeyKanzhelev Thanks for your feedback. I have made the changes. Please review and let me know if this looks good.

File: code-of-conduct.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    - Posting sexually explicit or violent material.
+    - Posting, or threatening to post, people's personally identifying information ("doxing").
+    - Insults, especially those using discriminatory terms or slurs.
+    - Behavior that could be perceived as sexual attention.
+    - Advocating for or encouraging any of the above behaviors.
+	
+- Understand disagreements: Disagreements, both social and technical, are useful learning opportunities. Seek to understand the other viewpoints and resolve differences constructively.
+- This code is not exhaustive or complete. It serves to capture our common understanding of a productive, collaborative environment. We expect the code to be followed in spirit as much as in the letter.
+
+
```
Comment (bogdandrutu) [inline code-of-conduct.md]: Remove some of the extra empty lines.

**Pull request merged**